From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-test

# DP: document conditional file expressions acting on the target of
# DP: symbolic links as well (except -h, -L).

# DP: document conditional file expressions acting on the target of
# DP: symbolic links as well (except -h, -L).
---
 builtins/test.def | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/builtins/test.def b/builtins/test.def
index da7870c..b6d5b81 100644
--- a/builtins/test.def
+++ b/builtins/test.def
@@ -64,6 +64,9 @@ File operators:
 
   FILE1 -ef FILE2  True if file1 is a hard link to file2.
 
+All file operators except -h and -L are acting on the target of a symbolic
+link, not on the symlink itself, if FILE is a symbolic link.
+
 String operators:
 
   -z STRING      True if string is empty.
